queue_job/models/queue_job_function.py

```diff
@@ -149,10 +149,12 @@ def _parse_retry_pattern(self):
         try:
             # as json can't have integers as keys and the field is stored
             # as json, convert back to int
-            retry_pattern = {
-                int(try_count): postpone_seconds
-                for try_count, postpone_seconds in self.retry_pattern.items()
-            }
+            retry_pattern = {}
+            for try_count, postpone_value in self.retry_pattern.items():
+                if isinstance(postpone_value, int):
+                    retry_pattern[int(try_count)] = postpone_value
+                else:
+                    retry_pattern[int(try_count)] = tuple(postpone_value)
```

Comment:
is this conversion to tuple necessary?

Reply:
If I leave that conversion out, the result in the computed value that is show in the `queue_job_function` form is a list. I believe it is a bit confusing if you write a tuple in the config and it results in a list, so I added that.